galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/TestRunner.java

```diff
@@ -542,12 +542,12 @@ private void runTestClassWrapper(TestClassWrapper testClassWrapper, TestRunManag
             return;
         }
```

Comment:
```
if (this.runType != RunType.Test && this.runType == RunType.SharedEnvironmentBuild) {
            return;
        }
        if (!runOk) {
            return;
        }
```
can be replaced by a single if statement, which doesn't explicitly return...

```
if ( runOk && 
     (this.runType != RunType.SharedEnvironmentBuild) ) {
  try...finally
}
```

Also `runOk` should be `isRunOk` by convention perhaps ?